src/traces/sankey/render.js

```diff
@@ -444,11 +444,11 @@ module.exports = function(svg, styledData, layout, callbacks) {
         .append('g')
         .classed('sankeyLinks', true)
         .style('fill', 'none')
-        .style('transform', linksTransform);
```

Comment:
I don't think we have an established workaround, either lint defining it or `var SVGElement = window.SVGElement` would be fine by me.